ingress/controllers/nginx-third-party/README.md

```diff
@@ -307,6 +308,93 @@ The route `/error` expects two arguments: code and format
 
 Using a volume pointing to `/var/www/html` directory is possible to use a custom error
 
+
+## Debug
+
+Using the flag `--v=XX` it is possible to increase the level of logging.
+In particular:
+- `--v=2` shows details using `diff` about the changes in the configuration in nginx
+
+```
+I0316 12:24:37.581267       1 utils.go:148] NGINX configuration diff a//etc/nginx/nginx.conf b//etc/nginx/nginx.conf
+I0316 12:24:37.581356       1 utils.go:149] --- /tmp/922554809  2016-03-16 12:24:37.000000000 +0000
++++ /tmp/079811012  2016-03-16 12:24:37.000000000 +0000
```

Comment:
please include an example of a simple tls ingress (that's supported right?):
https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce#tls

Just so we have a consistent experience. I'm ok doing this in another pr if you prefer.